src/schema-parser/raml-schema-parser.ts

```diff
@@ -0,0 +1,69 @@
+import { SchemaParser, ParseSchemaInput, ValidateSchemaInput } from "../schema-parser";
+import type { AsyncAPISchema, SchemaValidateResult } from '../types';
+import yaml from 'js-yaml';
+import * as lib from "webapi-parser";
+const wap = lib.WebApiParser;
+const r2j = require('ramldt2jsonschema');
+
+export function RamlSchemaParser(): SchemaParser {
+    return {
+      validate,
+      parse,
+      getMimeTypes,
+    }
+  }
+
+async function parse(input: ParseSchemaInput<unknown, unknown>): Promise<AsyncAPISchema> {
+    const message = (input.meta as any).message
+    try {
+        const payload = formatPayload(input.data);
+
+        // Draft 6 is compatible with 7.
+        const jsonModel = await r2j.dt2js(payload, 'tmpType', { draft: '06' });
+        const convertedType = jsonModel.definitions.tmpType;
+
+        message['x-parser-original-schema-format'] = input.schemaFormat || input.defaultSchemaFormat;
```

Comment:
Same as https://github.com/asyncapi/parser-js/pull/576#discussion_r943559156

Reply:
I wrote comment about that in PR for AVRO :) We will handle that in one place.